src/components/blocks/details/details.html

```diff
@@ -0,0 +1,38 @@
+<div class="image-overlay-detail">
```

Comment:
shouldn't all these "details" related files be named "image-overlay-detail"?
not just to keep with the standard structure, but why not, it only helps with clarity. it's also part of the whole name-spacing convention.

Reply:
all renamed now